src/mobile/src/ui/views/wallet/AdvancedSettings.js

```diff
@@ -165,6 +170,6 @@ const mapDispatchToProps = {
     generateAlert,
 };
 
-export default withNamespaces(['advancedSettings', 'settings', 'global'])(
+export default withNamespaces(['advancedSettings', 'settings', 'global', 'stateExport'])(
```

Comment:
I'm not sure if this needs to be added since the `exportState` string is in the `global` namespace, which is already specified